#goal definition
bool runPID
string intakeSide
---
#result definition
bool finished
---
#feedback
float64 result
